libnetwork/iptables/iptables.go

```diff
@@ -173,6 +173,24 @@ func (iptable IPTable) LoopbackByVersion() string {
 	return "127.0.0.0/8"
 }
 
+// SetupInternalICC is used to add or remove the internal interface from the firewalld zone ( ICC )
+// https://github.com/moby/libnetwork/issues/2647
+func (iptable IPTable) SetupInternalICC(bridgeIface string, include bool) error {
```

Comment:
Was there a reason to add this as a method on `iptable`? As (from the code below) it doesn't appear to be using `iptable` as part of the function 🤔 

(FWIW, the whole "iptable" interface is not ideal; it's trying to shoehorn / abstracting many things into an interface that's not always a good fit for the things done, so trying to prevent digging in further into adding more exported methods)